ILSpy.ReadyToRun/ILSpy.ReadyToRun.csproj

```diff
@@ -35,12 +35,12 @@
 
   <ItemGroup>
     <PackageReference Include="Iced" />
-    <PackageReference Include="ILCompiler.Reflection.ReadyToRun.Experimental" />
     <!-- ILCompiler.Reflection.ReadyToRun has dependencies on System.Reflection.Metadata and
```

Comment:
Is this comment still valid?

Reply:
Most likely... #3334 is also open in parallel